install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
```

Comment:
`/var/tmp` is theoretically also automatically cleaned up, FWIW.  http://0pointer.de/blog/projects/tmp.html further suggests that if you're working with large files you should avoid `/tmp`, but *shrug*.  There exist versions of Ubuntu that do the limited-size `/tmp` but I forget which ones.

Manual cleanup is maybe a little more annoying, but not really terrible:

```
WORK_DIR="$(mktemp -d)"
cleanup () {
  if [ -d "$WORK_DIR" ] ; then
    echo "Cleaning up "$WORK_DIR"
    rm -rf "$WORK_DIR"
  fi
}
trap cleanup EXIT
```

Then again, maybe you want to leave that folder around on failure so it can be inspected and maybe someone can figure out what went wrong, rather than it disappearing without a trace.